articles/140_topic_and_service_name_mapping.md

```diff
@@ -314,24 +351,6 @@ In ROS 2, topic and service names differ from ROS 1 in that they:
 
 This section lists concerns about the proposed design and alternatives that were considered.
 
-#### Substitution of the Namespace Delimiter
-
-Without the use of partitions, the namespace delimiter in ROS 2 topic and service names, a forward slash (`/`), will be replaced with double underscores (`__`).
-Note that as fully qualified ROS 2 topic and service names are absolute, there is always a leading forward slash (`/`).
-
-#### ROS to DDS Name Conversion Examples
-
-Here are some examples of translations between ROS topic names and the corresponding DDS topic names:
-
-| ROS Name          | Subsystem | DDS Name               |
-|-------------------|-----------|------------------------|
-| `/foo`            | Topic     | `rt__foo`              |
-| `/foo/bar`        | Topic     | `rt__foo__bar`         |
-| `/_foo/bar`       | Topic     | `rt___foo__bar`        |
-| `/foo/_bar`       | Topic     | `rt__foo___bar`        |
-| `/_foo/_bar`      | Topic     | `rt___foo___bar`       |
-| `/_foo/_bar/_baz` | Topic     | `rt___foo___bar___baz` |
-
```

Comment:
I replaced these with a section summarizing them below. I moved it down so it is closer to the sections that were alternatives to this (now alternative) proposal.